deployments/polygon/usdc/migrations/1681313581_more_comp.ts

```diff
@@ -0,0 +1,74 @@
+import { Contract } from 'ethers';
+import { DeploymentManager, migration } from '../../../../plugins/deployment_manager';
+import { calldata, exp, proposal } from '../../../../src/deploy';
+
+import { expect } from 'chai';
+
+const ERC20PredicateAddress = '0x40ec5B33f54e0E8A33A975908C5BA1c14e5BbbDf';
+const RootChainManagerAddress = '0xA0c68C638235ee32657e8f720a23ceC1bFc77C77';
+
+export default migration('1681313581_more_comp', {
```

Comment:
For posterity, the reason why the CI job failed to write `enacted` on the first proposal submission was because the migration name did not match the migration file name, so no file was find when trying to write `enacted`.